src/config.ts

```diff
@@ -586,6 +586,29 @@ export const ARNS_NAMES_CACHE_TTL_SECONDS = +env.varOrDefault(
   `${60 * 60}`, // 1 hour
 );
 
+export const ARNS_NAMES_CACHE_CIRCUIT_BREAKER_TIMEOUT_MS = +env.varOrDefault(
+  'ARNS_NAMES_CACHE_CIRCUIT_BREAKER_TIMEOUT_MS',
+  `${10 * 1000}`, // 10 seconds
```

Comment:
10 seconds per records request is more than reasonable

Reply:
Did you test what we see normally?

Reply:
i'm going off of grafana stats for cu.ardrive.io - the p99 there is < 2 seconds for a dry-run. will do similar tests against FWD CUs and add a comment